### ISSUE TITLE
When I use the test script, the memory of the GPU will gradually increase, and finally it will out of memory

### ISSUE THREAD

**Issue by moyans** [xingyizhou/CenterNet#496]
@xingyizhou

**Comment by xingyizhou** [comment]
I didn't meet this, please make sure you haven't changed any code. If that still doesn't work, please provide more detailed information.

**Comment by YohannaYin** [comment]
I have the same proble.when I test,it used 4 GPU at beginnning，but only the memory of ”gpu 0“ will increase with more pictures be tested.Suddently,it stopped due to out of memory.Hope you could help me,thanks.my gpu is 2080ti,every one is 16G.

**Comment by WangChen100** [comment]
Indeed, I have tried training with 1 or 2 GPU, it reminds me of the error "out of memory". 
It seems that if you do not train with 5GPU, the error appears.

**Comment by Cli98** [comment]
> I have the same proble.when I test,it used 4 GPU at beginnning，but only the memory of ”gpu 0“ will increase with more pictures be tested.Suddently,it stopped due to out of memory.Hope you could help me,thanks.my gpu is 2080ti,every one is 16G.

@YohannaYin 2080ti has only <11G memory, you should double check
